code/WordpressImportField.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+
+/**
+ * An upload fields which handles importing a wordpress xml file
+ * into silverstripe-blogger.
+ *
+ * @package silverstripe
+ * @subpackage bloggerwpimport
+ *
+ * @author Michael Strong <micmania@hotmail.co.uk>
+**/
+class WordpressImportField extends UploadField {
```

Comment:
I was planning on customising this with a wordpress logo in the drop area but never got around to it. Just FYI if you fancy polishing it off :p